cadence/transactions/create_forwarder.cdc

```diff
@@ -0,0 +1,47 @@
+import NonFungibleToken from "../contracts/NonFungibleToken.cdc"
+import MetadataViews from "../contracts/MetadataViews.cdc"
+import ExampleNFT from "../contracts/ExampleNFT.cdc"
+import NFTForwarding from "../contracts/utility/NFTForwarding.cdc"
+
+/// This transaction is what an account would run
+/// to set itself up to forward NFTs to a designated
+/// recipient's collection
+
+transaction(recipientAddress: Address) {
+
+    prepare(signer: AuthAccount) {
+        // Return early if the account already has an NFTForwarder
+        if signer.borrow<&NFTForwarding.NFTForwarder>(from: NFTForwarding.NFTForwarderStoragePath) != nil {
+            return
+        }
+
+        // Get Receiver Capability from the recipientAddress account
+        let receiverCapability = getAccount(recipientAddress)
+            .getCapability<&{NonFungibleToken.CollectionPublic}>(ExampleNFT.CollectionPublicPath)
+
+        // Make sure the CollectionPublic capability is valid before minting the NFT
+        if !receiverCapability.check() { panic("CollectionPublic capability is not valid!") }
+
+        // Create a new NFTForwarder resource
+        let forwarder <- NFTForwarding.createNewNFTForwarder(recipient: receiverCapability)
+
+        // save it to the account
+        signer.save(<-forwarder, to: NFTForwarding.NFTForwarderStoragePath)
+
+        // unlink existing Collection capabilities from PublicPath
+        if signer.getCapability(ExampleNFT.CollectionPublicPath)
+            .check<&{
+                NonFungibleToken.CollectionPublic,
+                ExampleNFT.ExampleNFTCollectionPublic,
+                MetadataViews.ResolverCollection
```

Comment:
I am not sure of MetadataViews part here

Reply:
This is modeled on how the collection is linked in [`cadence/transactions/setup_account.cdc`](https://github.com/onflow/flow-nft/blob/9073b1a538993274067babee4c4ba17d41698fd8/cadence/transactions/setup_account.cdc#L23)